source/_addonStore/models/status.py

```diff
@@ -115,6 +116,8 @@ def _displayStringLabels(self) -> Dict["AvailableAddonStatus", str]:
 			# Translators: Status for addons shown in the add-on store dialog
 			self.PENDING_ENABLE: pgettext("addonStore", "Enabled, pending restart"),
 			# Translators: Status for addons shown in the add-on store dialog
+			self.ENABLED: pgettext("addonStore", "Enabled"),
+			# Translators: Status for addons shown in the add-on store dialog
 			self.RUNNING: pgettext("addonStore", "Enabled"),

```

Comment:
should this be replaced with "Enabled (not running?)"

Alternatively, should `.RUNNING` have a display string of "Enabled (running)" instead of "Enabled"?

Reply:
> should the display string for .ENABLED be replaced with "Enabled (not running?)"

I like this one better personally.

Reply:
Instead, we've opted for a banner to warn users that NVDA was started with add-ons disabled